rootfs/setup_rootfs.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# TODO: apt keeps complaining about our locale not being set; we should probably set it properly
```

Comment:
I believe this is outdated now. It looks like locale/timezone are set up properly now.

Reply:
7a216c9